admin-dev/themes/new-theme/js/pages/order/view/order-product-edit.js

```diff
@@ -105,6 +105,9 @@ export default class OrderProductEdit {
   displayProduct(product) {
     this.productRowEdit = $(OrderViewPageMap.productEditRowTemplate).clone(true);
     this.productRowEdit.attr('id', `editOrderProduct_${this.orderDetailId}`);
+    this.productRowEdit.find('*[id]').each(function removeAllIds() {
```

Comment:
😆 
"A deleted ID is a good ID that does not mess with my code" 

<img src="https://image.shutterstock.com/image-photo/cool-cowboy-jacket-cigarette-over-260nw-781089601.jpg">

Reply:
I want to die